readme.md

```diff
@@ -127,7 +152,13 @@ Using jQuery, write an AJAX request to `http://tunr.com/artists` that would crea
 
 ```js
 // Your code starts here...
-
+$.ajax({
+  url: 'http://tunr.com/artists' + 'Resin Laying Deer Figurine, Gold'
+  type: 'post',
+  datatype: 'json'
+}).done(function{
+  alert('All Done!')
+})
```

Comment:
Close, but not quite. Remember that in a restful route, create doesn't have an in the URL. Instead you'd want something like the code below to include information /data about the artist you're trying to create:

```ruby
$.ajax({
  method: "POST",
  url: "http://tunr.com/artists",
  dataType: "json",
  data: {
    name: "Resin Laying Deer Figurine, Gold"
  }
}).done(function(){
  alert("All done!");
});
```